projects/rero/ng-core/src/lib/search-input/search-input.component.html

```diff
@@ -16,6 +16,7 @@
 -->
 <label for="search" class="font-weight-bold" *ngIf="displayLabel">{{ placeholder }}</label>
 <div class="input-group px-0">
+  <!-- TODO: remove autofocus -->
```

Comment:
Why TODO? Can you remove directive directly?

Reply:
This is actually an html attribute and not a directive. It may be useful on other views than the one that is impacted by this modifications right now. It would take time to check the behaviour of all inputs after removing this so Johnny proposed to do it after the release.